README.md

```diff
@@ -262,7 +271,7 @@ Add providers to allow CAPI to talk to various cloud providers.
 ### OpenStack
 
 * [Install the provider](charts/cluster-api-provider-openstack/README.md)
-* [Install a cluster](charts/cluster-api-cluster-openstack/README.md)
```

Comment:
Did you mean to change this?

Reply:
nope, looks like a fat finger. I've actually added that doc so it shoudl remain the same!

I'm going to push the chart updates and these docs cahgnes in a separate PR then I'll rebase this one with that once it's gone through. Let's make this a cleaner change.

Reply:
Corrected in the other PR